Tools/LyTestTools/tests/unit/test_o3de_material_canvas_test.py

```diff
@@ -0,0 +1,224 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+import unittest
+
+import pytest
+import unittest.mock as mock
+
+import ly_test_tools.o3de.material_canvas_test as material_canvas_test
+
+pytestmark = pytest.mark.SUITE_smoke
+
+
+class TestMaterialCanvasTestSuite(unittest.TestCase):
+
+    @mock.patch('ly_test_tools.o3de.editor_test_utils.kill_all_ly_processes')
+    def test_TestData_ValidAP_TeardownAPOnce(self, mock_kill_processes):
```

Comment:
These tests appear to be duplicates of tests which should only need to target the higher level interfaces of MultiTest, but are instead (re)implemented at the lower level. I don't/can't see anything in here that helps verify logic specific to handling the MaterialCanvas.

Reply:
It pretty much shares almost everything MaterialEditor uses except it has one new additional binding in `azlmbr.materialcanvas` though it is currently unused in this PR. I could remove them but then what would I add in its place? I guess I could check on the multi_test_framework unit tests to see.

Reply:
Correct, and I think it's highlighting how it's a bit overkill that the MaterialCanvasXYZTest classes need to reimplement so many parts of MaterialEditorXYZTest classes. There's a lot of OOP-boilerplate, and not as much code concisely adding what is unique.